index.html

```diff
@@ -3,21 +3,28 @@
     <head>
         <title>Coding Playlists</title>
         <link rel="stylesheet" href="style.css">
+        <script src="tabs.js"></script>
         <meta name="viewport" content="width=device-width, initial-scale=1.0">
         <meta name="description" content="Coding playlists to elevate focus, boost your productivity and enhance enjoyment on your coding journey.">
         <meta name="google-site-verification" content="YctCueJjkqNWfppJLoyMYRje4PTP2kzxwuf5C9mp6nI" />
     </head>
     <body>
         <header>
             <h1>Coding Playlists</h1>
-            <h4 id="contribution_link"><a href="https://github.com/gracekishino/coding_playlists" target="_blank">Contribute to Coding Playlists →</a></h4>
+            <button onclick="openPlayer('spotify')">Spotify</button>
+            <button onclick="openPlayer('youtube')">Youtube</button>
+            <button onclick="openPlayer('apple')">Apple</button>
+            <!-- <button onclick="window.location.href='https://github.com/gracekishino/coding_playlists">Contribute</button> -->
+            <!-- <h4 id="contribution_link"><a href="https://github.com/gracekishino/coding_playlists" target="_blank">Contribute to Coding Playlists →</a></h4> -->
         </header>
         <main>
             <div>
-                <p>Welcome to "Coding Playlists" – a laid-back space where the coding community comes together to share and discover chill soundtracks for productive coding vibes.</p>
+                <p>
+                Welcome to "Coding Playlists" – a laid-back space where the coding community comes together to share and discover chill soundtracks for productive coding vibes.</p>
+                <p>Ready to dive into the world of coding? pick a player from the <span id="nav_refer"><b>nav bar</b></span> and get CODING!!!</p>
             </div>
 
-            <section id="spotify">
+            <section id="spotify" style="display: none;" class="player_tab">
```

Comment:
This is so that spotify tab with be visible on page load.
```suggestion
            <section id="spotify" style="display: block;" class="player_tab">
```